src/seagulls/examples/space_shooter/_ship.py

```diff
@@ -48,6 +50,9 @@ class Ship(GameObject):
     _max_velocity: float
     _lasers: List[GameObject]
 
+    mixer.init()
+    _laser_sound = mixer.Sound("assets/sounds/laser-sound.ogg")
```

Comment:
these two lines are running as part of the definition of the class so we should move this into the constructor below. The `mixer.init()` probably goes somewhere else but I'm not sure where yet so just moving it into the constructor will be good enough for now :)

Reply:
moved both lines to the constructor!